src/info/manifest.c

```diff
@@ -383,6 +386,809 @@ manifestNewInternal(void)
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+
+                if (strEqZ(info->name, PG_PATH_PGSNAPSHOTS) && pgVersion >= PG_VERSION_92)
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+
+                if (strEqZ(info->name, PG_PATH_PGSTATTMP))
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+
+                if (strEqZ(info->name, PG_PATH_PGSUBTRANS))
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+            }
+
+            // Skip the contents of archive_status when online
+            if (buildData.online && strEq(buildData.manifestParentName, buildData.manifestWalName) &&
+                strEqZ(info->name, PG_PATH_ARCHIVE_STATUS))
+            {
+                FUNCTION_TEST_RETURN_VOID();
+                return;
+            }
+
+            // Recurse into the path
+            ManifestBuildData buildDataSub = buildData;
+            buildDataSub.manifestParentName = manifestName;
+            buildDataSub.pgPath = strNewFmt("%s/%s", strPtr(buildData.pgPath), strPtr(info->name));
+
+            if (buildData.dbPathExp != NULL)
+                buildDataSub.dbPath = regExpMatch(buildData.dbPathExp, manifestName);
+
+            storageInfoListP(
+                buildDataSub.storagePg, buildDataSub.pgPath, manifestBuildCallback, &buildDataSub, .sortOrder = sortOrderAsc);
+
+            break;
+        }
+
+        // Add files
+        // -------------------------------------------------------------------------------------------------------------------------
+        case storageTypeFile:
+        {
+            // Skip pg_internal.init since it is recreated on startup
+            if (strEqZ(info->name, PG_FILE_PGINTERNALINIT))
+            {
+                FUNCTION_TEST_RETURN_VOID();
+                return;
+            }
+
+            // Skip files in the root data path
+            if (strEq(buildData.manifestParentName, MANIFEST_TARGET_PGDATA_STR))
+            {
+                // Skip recovery files
+                if (((strEqZ(info->name, PG_FILE_RECOVERYSIGNAL) || strEqZ(info->name, PG_FILE_RECOVERYSIGNAL)) &&
+                        pgVersion >= PG_VERSION_12) ||
+                    ((strEqZ(info->name, PG_FILE_RECOVERYCONF) || strEqZ(info->name, PG_FILE_RECOVERYDONE)) &&
+                            pgVersion < PG_VERSION_12) ||
+                    // Skip temp file for safely writing postgresql.auto.conf
+                    (strEqZ(info->name, PG_FILE_POSTGRESQLAUTOCONFTMP) && pgVersion >= PG_VERSION_94) ||
+                    // Skip backup_label in versions where non-exclusive backup is supported
+                    (strEqZ(info->name, PG_FILE_BACKUPLABEL) && pgVersion >= PG_VERSION_96) ||
```

Comment:
It was an oversight.  If somebody is running an exclusive backup side-by-side with our non-exclusive backup, we certainly don't want to copy their `backup_label`.  I think in Perl it would just get overwritten with our `backup_label` at the end of the backup, but C definitely won't work that way.

It's an edge case, but worth handling.